src/Config/ObjectModel/HostOptions.cs

```diff
@@ -32,7 +32,7 @@ public record HostOptions
     public AuthenticationOptions? Authentication { get; init; }
 
     [JsonPropertyName("mode")]
-    public HostMode Mode { get; init; }
+    public HostMode Mode { get; set; }
```

Comment:
Why do we need to change it to `set`?

Reply:
This is because `init` will only allow the value to be changed in the object initializer. In order to overwrite this value from another class, such as when we hot-reload, we need to have a mutator that is public, which is what `set` provides.